atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/cleaner/ImmutableTimestampSupplier.java

```diff
@@ -43,23 +43,21 @@
                 TimeUnit.MILLISECONDS);
     }
 
-    private final RemoteLockService lockService;
     private final TimestampService timestampService;
-    private final LockClient lockClient;
+    private final MonitoredMinLockedVersionProvider minLockedVersionProvider;
```

Comment:
would prefer not to modify this class (see other comment about passing this to the monitor)